lib/deploy/lifecycleHooks.js

```diff
@@ -1,43 +1,78 @@
 'use strict';
 
+var _ = require('lodash');
 var RSVP = require('rsvp');
 var utils = require('../utils');
 var chalk = require('chalk');
-var process = require('child_process');
+var cprocess = require('child_process');
 var FirebaseError = require('../error');
+var getProjectId = require('../getProjectId');
+var logger = require('../logger');
+
+
+function runCommand(command, options) {
+  return new Promise(function(resolve, reject) {
+    var child = cprocess.spawn(command, _.assign({}, options, {
+      shell: true,
+      stdio: [0, 1, 2], // Inherit STDIN, STDOUT, and STDERR
+      env: process.env
```

Comment:
I believe this line is unnecessary, it should automatically set the spawned process' env to the same as the parent. Have you tried taking it out?

Reply:
Removed